google/cloud/bigquery_storage_v1/writer.py

```diff
@@ -60,13 +60,35 @@ def _wrap_as_exception(maybe_exception) -> Exception:
     return Exception(maybe_exception)
 
 
+def _clear_proto3_fields(
```

Comment:
nit: probably a better name here, since this function already does more than clear proto3 fields.  make it clear its copying and normalizing the template?

Reply:
Thanks, I renamed it to `_process_request_template()`.